POLIO-691 Speed up calendar map (campaign shapes) :fire:

olethanh opened the pull request:
It was the shapes endpoint that returned the geojson that got very slow. We speed it up From 11 minutes to 9seconds by now used the simplified_geom instead of the geom.

This code is only a port of the fix, we also need to manually run the following command manually to recalculate the simplified_geom on iaso_orgunit for the newly added org unit since the last time it was ran.

update iaso_orgunit set simplified_geom = ST_Simplify(geom::geometry, 0.01, true);

We need a more permanent fix in the futures to keep them up to date, but this is outside the scope of this Hotfix PR.

Explain what problem this PR is resolving

## Self proof reading checklist

- [ ] Did I use eslint and black formatters
- [ ] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] Did I add translations
- [ ] My migrations file are included
- [ ] Are there enough tests
## Changes

Explain the changes that were made. The idea is not to list exhaustively all the changes made (Github already provides a full diff), but to help the reviewers better understand:
- which specific file changes go together, e.g: when creating a table in the front-end, there usually is a config file that goes with it
- the reasoning behind some changes, e.g: deleted files because they are now redundant
- the behaviour to expect, e.g: tooltip has purple background color because the client likes it so, changed a key in the API response to be consistent with other endpoints


## How to test

Explain how to test your PR.
If a specific config is required explain it here (dataset, account, profile, ...)

## Print screen / video

Upload here print screens or videos showing the changes

## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here

madewulf commented on docker-compose.yml:
```diff
@@ -29,11 +29,11 @@ services:
       IASO_ENVIRONMENT: "development"
       RDS_HOSTNAME: db
-      RDS_DB_NAME: iaso
+      RDS_DB_NAME: ${RDS_DB_NAME:-iaso}
```
So far, default values were set in settings.py. Isn't that enough? 

It might not be, since webpack is picking on these variables now.

madewulf replied:
Example: DB_NAME = os.environ.get("RDS_DB_NAME", "postgres")

olethanh replied:
I set it like this because there was a value before and I didn't want it to break existing set up if people didn't have a .env file. Also docker compose was showing an ugly warning. But I can change it if you prefer.

olethanh committed:
POLIO-691 Speed up calendar map (campaign shapes) :fire:

It was the shapes endpoint that returned the geojson that got very slow.
We speed it up From 11 minutes to 9seconds by now used the
simplified_geom instead of the geom.

This code is only a port of the fix, we also need to manually run the following command manually
to recalculate the simplified_geom on iaso_orgunit for the newly added
org unit since the last time it was ran.

update iaso_orgunit set simplified_geom = ST_Simplify(geom::geometry, 0.01, true);

We need a more permanent fix in the futures to keep them up to
date, but this is outside the scope of this Hotfix PR.
